src/Stratis.Bitcoin.Features.Api/Stratis.Bitcoin.Features.Api.csproj

```diff
@@ -29,9 +29,21 @@
     <PackageReference Include="System.Reactive" Version="3.1.1" />
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
   </ItemGroup>
-
+      
   <ItemGroup>
     <ProjectReference Include="..\Stratis.Bitcoin\Stratis.Bitcoin.csproj" />
+    <ProjectReference Include="..\NBitcoin\NBitcoin.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="dashboard_static_files\" />
+    <Folder Include="Controllers\" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="dashboard_static_files\*.*" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="dashboard_static_files\*.*" />
```

Comment:
This is where we mark the directory's contents as embedded resources. Probably won't work on subfolders?